package.json

```diff
@@ -26,16 +26,17 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@aeternity/aepp-sdk": "^11.0.1",
+    "@aeternity/aepp-sdk": "^13.0.0-alpha.0",
+    "@aeternity/aepp-sdk-11": "github:aeternity/aepp-sdk-js#tags/v11.0.1",
```

Comment:
Keeping SDK 11 only for middleware swagger client generation only. Shall be removed after moving to pre-generating middleware swagger API clients